### PR TITLE
hyper-zoo Dockerfile multi-stage build

### PR DESCRIPTION
**Pull request by zzti-bsj**
## hyper-zoo multi-stage Dockerfile

The following commits are about Dockerfile multi-stage build.

### CODE REVIEW

File: docker/hyperzoo/Dockerfile [hunk 1]
```diff
@@ -1,23 +1,94 @@
-FROM ubuntu:18.04
-
```
Comment (glorysdj) [inline docker/hyperzoo/Dockerfile:2]: we dont need to modify this dockerfile

File: pyzoo/zoo/examples/tensorflow/tfpark/estimator/estimator_inception.py [hunk 2]
```diff
@@ -75,7 +75,8 @@ def model_fn(features, labels, mode, params):
 
     estimator = TFEstimator.from_model_fn(model_fn,
                                           params={"image_path": option.image_path,
-                                                  "num_classes": option.num_classes})
+                                                  "num_classes": option.num_classes,
+                                                  "batch_size": option.num_classes})
```
Comment (glorysdj) [inline pyzoo/zoo/examples/tensorflow/tfpark/estimator/estimator_inception.py:79]: batch_size should not use option.num_classes

File: pyzoo/zoo/examples/tensorflow/tfpark/estimator/estimator_inception.py [hunk 3]
```diff
@@ -49,7 +49,7 @@ def input_fn(mode, params):
             feature_set = feature_set.transform(ImageFeatureToSample())
             dataset = TFDataset.from_feature_set(feature_set,
                                                  features=(tf.float32, [224, 224, 3]),
-                                                 labels=(tf.int32, [1]), batch_size=16)
+                                                 labels=(tf.int32, [1]), batch_size=int(params["batch_size"]))
```
Comment (glorysdj) [inline pyzoo/zoo/examples/tensorflow/tfpark/estimator/estimator_inception.py:52]: if no batch_size as params, it should take 16 as default value

File: pyzoo/zoo/examples/tensorflow/tfpark/tf_optimizer/evaluate.py [hunk 4]
```diff
@@ -28,12 +28,12 @@
...
 
 
-def main(data_num):
+def main(options, data_num):
 
     sc = init_nncontext()
 
     # get data, pre-process and create TFDataset
-    (images_data, labels_data) = mnist.read_data_sets("/tmp/mnist", "test")
+    (images_data, labels_data) = mnist.read_data_sets(options.data_path, "test")
```
Comment (glorysdj) [inline pyzoo/zoo/examples/tensorflow/tfpark/tf_optimizer/evaluate.py:36]: split this to another pr?

Comment (zzti-bsj) [reply pyzoo/zoo/examples/tensorflow/tfpark/tf_optimizer/evaluate.py:36]: ok, I' will modify this.

**Comment by glorysdj** [comment]
can this be closed?
